coffea-dask/Dockerfile.almalinux8

```diff
@@ -22,6 +22,9 @@ RUN mamba install --yes python=${PYTHON_VERSION} \
      && mamba env update --file /environment.yaml \
      && mamba clean -y --all
 
+# FIXME: # disable microarches for awkward-cpp
+RUN mamba remove awkward awkward-cpp --force && pip install awkward
```

Comment:
Not sure if a `-y` is needed to do `mamba remove` here and on the other Dockerfile as well. It may work as is.

Reply:
Let see in GH CI

Reply:
It sadly removes too many packages:
```
[cms-jovyan@jupyter-oksana-2eshadura-40cern-2ech ~]$  mamba remove awkward awkward-cpp --force
Removing specs: ['awkward', 'awkward-cpp']
Transaction

  Prefix: /usr/local

  Removing specs:

   - awkward
   - awkward-cpp


  Package            Version  Build            Channel          Size
──────────────────────────────────────────────────────────────────────
  Remove:
──────────────────────────────────────────────────────────────────────

  - coffea          2024.8.1  pyhd8ed1ab_0     conda-forge     129kB
  - uproot            5.3.10  ha770c72_0       conda-forge       7kB
  - toml              0.10.2  pyhd8ed1ab_0     conda-forge      18kB
  - numba             0.60.0  py310h5dc88bb_0  conda-forge       4MB
  - dask-histogram  2024.3.0  pyhd8ed1ab_0     conda-forge      26kB
  - dask-awkward    2024.7.0  pyhd8ed1ab_0     conda-forge      70kB
  - uproot-base       5.3.10  pyhd8ed1ab_0     conda-forge     236kB
  - llvmlite          0.43.0  py310h4c7c693_0  conda-forge       3MB
  - cachetools         5.5.0  pyhd8ed1ab_0     conda-forge      15kB
  - cramjam            2.8.3  py310hcb5633a_0  conda-forge       2MB
  - awkward            2.6.7  pyhd8ed1ab_0     conda-forge     406kB
  - libllvm14         14.0.6  hcd5def8_4       conda-forge      31MB
  - awkward-cpp           37  py310haa687a3_0  conda-forge     511kB

  Summary:

  Remove: 13 packages

  Total download: 0 B

──────────────────────────────────────────────────────────────────────
```

Reply:
@ikrommyd basically the solution will be to install coffea from pip...which is not my favorite solution.

Reply:
That's strange. Conda docs specify "If you wish to skip this dependency checking and remove just the requested packages, add the '--force' option."